.gitlab-ci.yml

```diff
@@ -143,7 +158,7 @@ Packer:
   stage: test
   extends: .terraform
   rules:
-    - if: '$CI_PIPELINE_SOURCE != "schedule"'
+    - !reference [.ga/eus_rules_all, rules]
```

Comment:
shouldn't this also include    `- !reference [.upstream_rules_all, rules]` ?

Reply:
hmmm, I don't really know what this job does but the old rule means "execute on PRs" (e.g. the job wasn't scheduled via pipeline schedules) while the new one means the opposite: "execute on this particular scheduled pipeline"